common/types/layer.go

```diff
@@ -14,6 +14,7 @@ const (
 	LayerIDSize = 4
 
 	// BootstrapBeacon is the hex value of the beacon used during genesis.
+	// TODO: make this a config.
 	BootstrapBeacon = "0x333c04dd151a2a6831c039cb9a651df29198be8a04e16ce861d4b6a34a11c954" // sha256("bootstrap")
```

Comment:
```suggestion
	BootstrapBeacon = "0x333c04dd151a2a6831c039cb9a651df29198be8a04e16ce861d4b6a34a11c954" // blake3("bootstrap")
```

Reply:
but thats not true. that value is generated via sha256. in any case, that constant needs to go to a config

Reply:
OK, I didn't know. That's a bit confusing. Why is it the only thing that uses sha256?

Reply:
it was generated when everything else uses sha256. now we are changing to blake3. i just don't see a point of regenerating this with blake3 when it really should live in the config. hence the TODO.